### DISCUSSION TITLE
Welcome to GCAM discussions!

### DISCUSSION THREAD

**Discussion by crvernon** [JGCRI/gcam-core#163] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Check out the full [GCAM discussion forum](https://github.com/JGCRI/gcam-core/discussions) where you can start a thread using one of the following categories to help you get where you want to go more quickly:

- **Announcements** which provide updates from the maintainers of GCAM. 
- **Q&A** to ask the GCAM community for help.  This gives you the ability to suggest answers and upvote those that really fit the bill.
- **Show and tell** to show off your new GCAM research with the community!
- **Installing and running GCAM** to provide tips and tricks for installing and running GCAM.
- **Modifying GCAM** to get/give community advice on how to modify GCAM to suit your research needs.
- **Understand my results** to discuss your results and how to interpret them.

Say 👋 with a comment below that gives an introduction of yourself and tell us about what you do with this community.

**Comment by Curtis-dai** [comment]
I am from China and a green hand studying the gcam.i want to know how to view the results of gcam whose format is basex.i can not use the model interface to open the results.

**Reply by CUMTB-jzq19931285881** [reply]
wechat:19931285881
